README.md

```diff
@@ -128,6 +128,59 @@ render json: @posts, each_serializer: PostPreviewSerializer
 render json: @posts, serializer: CollectionSerializer, each_serializer: PostPreviewSerializer
 ```
 
+### Serialization Scope
+
+The [`serialization_scope` method](https://github.com/rails-api/active_model_serializers/blob/d02cd30fe55a3ea85e1d351b6e039620903c1871/lib/action_controller/serialization.rb#L13-L20)
+ allows you to include in the serializer access to an external method.
+By default, it is the `:current_user` method on the controller.
+
+Upon instantiation, the serializer defines the instance methods:
+- `scope` (with the value of `options[:scope]`) and
+- a method named the value of `options[:scope_name]`
+  (e.g. `current_user`, if not already present), that calls 'scope'.
+
+In the usual case, the serializer instance method has the same name as the controller method.
+e.g. `serializer.current_user` calls `controller.current_user`. The method's name
+will always be the `serialization_scope` defined in the controller, but may be
```

Comment:
This sentence is a bit hard to understand (at least for me). What is "the `serialization_scope`"? The parameter to the (class) method call `serialization_scope :foo`? (i.e. `:foo` in this case) I guess not as you talk about the controller.